ruby/tests/basic.rb

```diff
@@ -17,7 +17,6 @@ module BasicTest
     add_message "BadFieldNames" do
       optional :dup, :int32, 1
       optional :class, :int32, 2
-      optional :"a.b", :int32, 3
```

Comment:
Why remove this test?

Reply:
The new code doesn't allow this message type to be built. It errors out at this point instead of creating a broken message and descriptor.